src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -359,6 +359,12 @@ class FilterPopup extends React.Component {
         </ul>
       </div>
     );
+    const dateInputError = _map(this.state.raisedErrors, (item) => {
```

Comment:
Do you think it is necessary to create a new array with the error objects in there and then pass the `.length` of that array to `FieldsetDate`? It's doing too much. There's a different underscore method you can use to get a true/false value.

Besides, this is the error in the console I see:
<img width="825" alt="screen shot 2017-10-02 at 8 13 35 pm" src="https://user-images.githubusercontent.com/1280564/31104926-38d756d2-a7ae-11e7-98bf-80345fbeac4e.png">

It is telling you that you're passing a number (in this component) but that it should be a boolean, which you defined in the `FieldsetDate` component.

Reply:
hmmm `submitError={dateInputError.length}` should be  `submitError={dateInputError.length > 0}` in this case I think.

Reply:
That's correct. You _could_ do that but it's also better to just get the true/false value directly into `dateInputError` instead of creating a new array and checking for it's length.